src/settings/BatchGroupConfigurationSettings/ExportVoucherButton/ExportVoucherButton.js

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import saveAs from 'file-saver';
+
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+import { IconButton } from '@folio/stripes/components';
+import { useShowCallout } from '@folio/stripes-acq-components';
+
+import { BATCH_VOUCHERS_API } from '../../../common/constants';
+
+const ExportVoucherButton = ({ batchVoucherId, format, stripes }) => {
+  const showCallout = useShowCallout();
+  const downloadBatchVouchers = useCallback(
+    async () => {
+      try {
+        const httpHeaders = {
+          'X-Okapi-Tenant': stripes.okapi.tenant,
+          'X-Okapi-Token': stripes.store.getState().okapi.token,
+          'Accept': format.toLowerCase(),
```

Comment:
IMHO, it's a bad practice. Maybe it's better to create a map, where keys will be constants of batch export config format, and keys - corresponding HTTP headers

Reply:
imo it's ok to use array with all possible formats (xml and json here?)